relentless/optimize/method.py

```diff
@@ -177,41 +193,67 @@ def __init__(self, abs_tol, max_iter):
         self.abs_tol = abs_tol
         self.max_iter = max_iter
 
-    def find(self, objective, start, end):
+    def find(self, objective, start, end, grad_scale=1.0):
```

Comment:
I would prefer to call this parameter `scale` rather than `grad_scale`. Really, you are rescaling the **variables**, and the consequence is seen in the gradient.